src/features/winterStorageAreaView/utils.ts

```diff
@@ -55,6 +55,68 @@ export const getIndividualWinterStorageArea = (
   };
 };
 
-export const getCustomers = (data: INDIVIDUAL_WINTER_STORAGE_AREA | undefined): WinterStorageCustomer[] => {
-  return [{ id: 'placehold', name: 'placeholder' }];
+export const getWinterStoragePlaces = (data: INDIVIDUAL_WINTER_STORAGE_AREA | undefined): WinterStoragePlace[] => {
+  if (!data?.winterStorageArea?.properties) {
+    return [];
+  }
+  return data?.winterStorageArea?.properties?.sections.edges
+    .reduce<SectionProperties[]>((acc, sectionEdge) => {
+      if (!sectionEdge?.node?.properties) {
+        return acc;
+      }
+      return [...acc, sectionEdge.node.properties];
+    }, [])
+    .map<WinterStoragePlace[]>(
+      (sectionProperties) =>
+        sectionProperties.places.edges.reduce<WinterStoragePlace[]>((acc, placeEdge) => {
+          if (!placeEdge?.node) {
+            return acc;
+          }
+          const leases = placeEdge?.node?.leases?.edges.reduce<Lease[]>((acc, leaseEdge) => {
```

Comment:
The function is quite big, would it make sense to extract this piece as separate function (`getLeases`) ?

Reply:
Done.